### PR TITLE
Feat payment form

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: app/client/templates/exhibitor/payment.html [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+      <form id="pay" enctype="application/x-www- form-urlencoded" action="" method="post">
+        <input type="hidden" value="{{ _id }}" id="application-id">
+        <div class="row">
+          <div class="col s12 m6">
+            <label for="dollars" class="required">{{_ "exhibitor-pay-dollars"}}</label>
+            <input type="number" name="dollars" id="dollars" size="5" placeholder="100" required>
+          </div>
+          <div class="col s12 m6">
+            <label for="cents" class="required">{{_ "exhibitor-pay-cents"}}</label>
+            <input type="number" name="cents" id="cents" size="2" placeholder="00" required>
```
Comment (mixcas) [inline app/client/templates/exhibitor/payment.html]: `size` attr doesn't work for `type="number"`
should be `maxlenght`

Comment (mixcas) [inline app/client/templates/exhibitor/payment.html]: `The size attribute specifies the visible width, in characters, of an <input> element.`

File: app/client/router/routerApplications.js [hunk 2]
```diff
@@ -10,14 +10,14 @@ Router.map(function() {
 
         var userApplication = Applications.findOne({}, {'status': 1, 'extend': 1});
 
-        if (moment().isAfter(Meteor.settings.public.applicationDeadline)) {
-        
+        if (moment().isAfter(Meteor.settings.public.applicationDeadline) && userApplication.status !== 'approved') {
```
Comment (varffs) [inline app/client/router/routerApplications.js:13]: The next if also checks if after deadline. Could these 3 conditionals just be in 1?

File: app/client/templates/exhibitor/payment.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
+Template.payment.created = function () {
+  //
+};
+
+Template.payment.helpers({
+  //
+});
+
+Template.payment.rendered = function () {
+  $('select').material_select();
```
Comment (varffs) [inline app/client/templates/exhibitor/payment.js:10]: This is supposed to have the template prefix thing

File: app/client/templates/exhibitor/payment.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+  Session.set('paymentErrors', 0);
```
Comment (varffs) [inline app/client/templates/exhibitor/payment.js:12]: Where is the check of this value to stop creating errors and tell user to email directly?

Comment (varffs) [inline app/client/templates/exhibitor/payment.js:12]: Needs this tracker auto run and markup for the alert: https://github.com/interglobalvision/app-material-fair-com/blob/master/app/client/templates/application/signed.js#L8

**Pull request merged**